adodbapi/adodbapi.py

```diff
@@ -24,7 +24,6 @@
 DB-API 2.0 specification: http://www.python.org/dev/peps/pep-0249/
 
 This module source should run correctly in CPython versions 2.7 and later,
```

Comment:
I'm trying to keep specifically CPython2 code cleanup to its own PR (https://github.com/mhammond/pywin32/pull/1990), and splitting things up in smaller bites. Otherwise I end up with a ton of changes if I remove/update python2 specific code (ie, that PR), and when something break (see failing test) it's harder to find which change did it. Comments like these also help find said obsolete code.

That said, this comment doesn't point to specific Python2 code, so if you want me to update it in this here PR, I can do that. (but at least wanted to explain why I didn't remove it in the first place)